t/issue/948.t

```diff
@@ -0,0 +1,81 @@
+use strict;
+use warnings;
+
+use Rex::Virtualization;
+use Test::More tests => 6;
+use Data::Dumper;
+use Rex::Helper::Run;
+use Rex::Commands::File;
+use Rex::Commands::Fs;
+use Rex::Commands::Run;
+
+my $image_format = "raw";
+
+sub get_image_format { return $image_format; }
+
+my $count_file = 0;
+my $count_exec = 0;
+
+no warnings 'redefine';
```

Comment:
1. Should this `warnings` override be in effect to the end of the file? Can we put it inside a block, or even better, put a short `use warnings;` later where the override should end?